.gitignore

```diff
@@ -2,15 +2,19 @@
 .DS_Store
 examples/**/target/
 examples/9
-.vscode
```

Comment:
@kevaundray why was this ignored? We want to allow it to recommend plugins and default some settings in the workspace.

Reply:
I think this was a part of my default gitignore or a gitignore template